docs/release_process.rst

```diff
@@ -0,0 +1,175 @@
+=====================
+LORE Release Workflow
+=====================
+
+Quick Start
+-----------
+
+prerequisites:
+  - ``git-release-notes`` - https://www.npmjs.com/package/git-release-notes
+  - ``git-flow`` - https://github.com/nvie/gitflow
+  - You've configured Git-Flow to use LORE branch naming scheme.
+
+These instructions presume you're releasing 0.10.0.
+
+.. code-block:: bash
+
+    git fetch
+    git checkout master
+    git pull
+    git flow release start 0.10.0
+    vi lore/settings.py # change version number
+    git-release-notes v0.9.0..master util/release_notes_rst.ejs
+    vi RELEASE.rst # paste results of prior command at top of file, correcting the release number
+    git add RELEASE.rst lore/settings.py
+    git commit -m "Release 0.10.0"
+    # verify that the Travis-ci build succeeds
+    # Wait for all checkboxes to be checked
+    # Check with Dev-Ops buddy whether migrations need to be run for this release
+    git flow release publish 0.10.0
+    git push --tags
+    # Send release notification to LORE mailing list - odl-lore-announce@mit.edu
+
+Annotated Instructions
+----------------------
+
+.. code-block:: bash
+
+    git fetch
+    git checkout master
+    git pull
+    git flow release start 0.10.0
+
+Update the ``VERSION`` number in ``lore/settings.py`` following Semantic
+numbering practice.
+
+.. code-block:: bash
+
+    vi lore/settings.py # change version number
+
+Create the release notes with a utility called ``git-release-notes``.
+https://www.npmjs.com/package/git-release-notes  Its output is written
+to the console.
+
+``git-release-notes`` takes two command line arguments.  The first argument
+takes a range of git references to determine which commits to include in
+the release.  In the example given it is all commits from the last release
+to what is in ``master``.
+
+The second command argument specifies a template to format the output.
+LORE has two templates.
+
+- ``util/release_notes_rst.ejs`` formats the output as
+  reStructuredText for inclusion in ``RELEASE.rst``.
+
+- ``util/release_notes.ejs`` formats the output as to precede each commit
```

Comment:
I think this is clearer:

"... formats the output as markdown so as to precede... "